gnarails.gemspec

```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "rails", "~> 5.2.1"
   spec.add_dependency "thor"
 
-  spec.add_development_dependency "bundler", "~> 1.16"
+  spec.add_development_dependency "bundler", "~> 2.0.1"
```

Comment:
Well, that's a pretty good reason this wouldn't work while developing, though interesting that this would be causing issues with the shipped version of the gem as well. @alxjrvs, when you initially reported this, were you using the gem itself, or were you pulling from master locally?